Movie_MVP/Movie_MVP/DescriptionModule/DescriptionPresenter/DescriptionPresenter.swift

```diff
@@ -40,12 +43,24 @@ final class DescriptionPresenter: DescriptionPresenterProtocol {
         router.routeToWebView(imdbId: imdbId)
     }
 
+    private func loadMovieFromRealm() {
```

Comment:
```suggestion

// MARK: - Private Methods
    private func loadMovieFromRealm() {
```
и приватные методы ниже чем паблик
и кажется название не подходит? у тебя же загрузка не только из рилма, из сети тоже